lib/registries/addon/page-not-found/template.hbs

```diff
@@ -17,12 +17,19 @@
                             {{t 'not_found.registries.contact'}}
                         </a>
                     </p>
-                    {{#link-to 'index'
+                    <LinkTo
+                        @route='index'
                         class='btn btn-primary m-t-md'
-                        invokeAction=(action 'click' 'link' 'Page not found - Go to Index' target=this.analytics)
-                    }}
+                        invokeAction={{
```

Comment:
Looks like this can be replaced with a `data-analytics-name="Page not found - Go to Index"`  as well